test/integration/stateboard_test.lua

```diff
@@ -35,101 +36,114 @@ g.after_each(function()
     fio.rmtree(g.datadir)
 end)
 
-local function connect(srv)
-    return netbox.connect(srv.net_box_port, srv.net_box_credentials)
+local function create_client(srv)
+    return stateboard_client.new({
+        uri = 'localhost:' .. srv.net_box_port,
+        password = srv.net_box_credentials.password,
+        call_timeout = 1,
+    })
```

Comment:
Would you like to make it `return stateboard_client.new():get_session()`? Because below they're always used together.